### PR TITLE
Update Key Visualizer doc

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Update Key Visualizer doc.
This PR must be merged after https://github.com/pingcap/docs/pull/2655.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3250
- Other reference link(s):

### CODE REVIEW

File: dashboard/key-visualizer-monitoring-tool.md [hunk 1]
```diff
@@ -1,34 +1,33 @@
 ---
-title: Key Visualizer
-summary: Learn how to monitor traffic using the Key Visualizer tool.
+title: Key Visualizer Page
+summary: Learn how to monitor traffic using the Key Visualizer page.
 category: how-to
+aliases: ['/docs-cn/dev/key-visualizer-monitoring-tool/']
```
Comment (TomShawn) [inline dashboard/key-visualizer-monitoring-tool.md:5]:
```suggestion
aliases: ['/docs/dev/key-visualizer-monitoring-tool/']
```

**Comment by yikeke** [comment]
Please help review this PR, thanks~ @CaitinChen

File: dashboard/key-visualizer-monitoring-tool.md [hunk 2]
```diff
@@ -1,34 +1,33 @@
 ---
-title: Key Visualizer
-summary: Learn how to monitor traffic using the Key Visualizer tool.
+title: Key Visualizer Page
+summary: Learn how to monitor traffic using the Key Visualizer page.
```
Comment (CaitinChen) [inline dashboard/key-visualizer-monitoring-tool.md:3]:
```suggestion
summary: Learn how to use Key Visualizer to monitor traffic.
```

File: dashboard/key-visualizer-monitoring-tool.md [hunk 3]
```diff
@@ -69,7 +68,25 @@ A TiDB cluster might have up to hundreds of thousands of Regions. It is difficul
...
 This section introduces how to use Key Visualizer.
 
-![Toolbar](/media/dashboard/keyvisualizer/toolbar.png)
+### Settings
+
+To use the Key Visualizer page for the first time, you need to manually enable this feature on the **Settings** page. Follow the page guide and click **Open Settings** to open the settings page:
+
+![Feature disabled](/media/dashboard/dashboard-keyviz-not-enabled.png)
+
+After this feature is enabled, you can open the settings page by clicking the **Settings** icon at the upper right corner:
```
Comment (CaitinChen) [inline dashboard/key-visualizer-monitoring-tool.md:77]:
```suggestion
After this feature is enabled, you can open the settings page by clicking the **Settings** icon in the upper right corner:
```

File: dashboard/key-visualizer-monitoring-tool.md [hunk 4]
```diff
@@ -69,7 +68,25 @@ A TiDB cluster might have up to hundreds of thousands of Regions. It is difficul
...
+
+![Settings icon](/media/dashboard/dashboard-keyviz-settings-button.png)
+
+The settings page is shown as follows:
+
+![Settings page](/media/dashboard/dashboard-keyviz-settings.png)
+
+Set whether to start the data collection through the switch, and click **Save** to take effect. After enabling the feature, you can see that the toolbar is available:
```
Comment (CaitinChen) [inline dashboard/key-visualizer-monitoring-tool.md:85]:
```suggestion
Set whether to start data collection through the switch, and click **Save** to take effect. After enabling the feature, you can see that the toolbar is available:
```

File: dashboard/key-visualizer-monitoring-tool.md [hunk 5]
```diff
@@ -1,34 +1,33 @@
...
+![Access Key Visualizer](/media/dashboard/dashboard-keyviz-access.png)
+
+* Visit <http://127.0.0.1:2379/dashboard/#/keyviz> in your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.
 
 > **Note:**
 >
-> + If multiple PD instances exist, you can use any PD instance address to access TiDB Dashboard.
-> + By default, `PDPort` is `2379`. If you have modified the corresponding PD parameter during deployment, use the modified PD port.
-> + You need to use the `root` account of TiDB to log into TiDB Dashboard.
+> The Key Visualizer feature is disabled by default. If you use this feature for the first time, You need to manually enable it. To enable this feature, follow the interface guide or see the [Settings](#settings) section of this document.
```
Comment (HunDunDM) [inline dashboard/key-visualizer-monitoring-tool.md:24]: This feature is currently enabled by default, please delete this part.

File: dashboard/key-visualizer-monitoring-tool.md [hunk 6]
```diff
@@ -115,39 +139,39 @@ To regain a heatmap based on the current time, click the **Refresh** button. If
...
+![Copy Bucket details](/media/dashboard/dashboard-keyviz-tooltip-copy.png)
 
 ## Common heatmap types
 
 This section shows and interprets four common types of heatmap in Key Visualizer.
 
 ### Evenly distributed workload
 
-![Balanced ](/media/dashboard/keyvisualizer/well_dist.png)
+![Balanced ](/media/dashboard/dashboard-keyviz-well-dist.png)
```
Comment (HunDunDM) [inline dashboard/key-visualizer-monitoring-tool.md:154]:
```suggestion
![Balanced](/media/dashboard/dashboard-keyviz-well-dist.png)
```

File: dashboard/key-visualizer-monitoring-tool.md (hunk 5 above)
Comment (TomShawn) [reply dashboard/key-visualizer-monitoring-tool.md:24]: Deleted.

**Comment by TomShawn** [comment]
@HunDunDM Comment addressed, PTAL again.

**Pull request merged**